src/PlusDataCollection/Atracsys/AtracsysTracker.h

```diff
@@ -0,0 +1,198 @@
+/*=Plus=header=begin======================================================
+Program: Plus
+Copyright (c) Laboratory for Percutaneous Surgery. All rights reserved.
+See License.txt for details.
+=========================================================Plus=header=end*/
+
+#ifndef __AtracsysTracker_h
+#define __AtracsysTracker_h
+
+#include <string>
+#include <vector>
+
+// Atracsys includes
+#include "ftkErrors.h"
+#include "ftkEvent.h"
+#include "ftkInterface.h"
+#include "ftkOptions.h"
+#include "ftkPlatform.h"
+#include "ftkTypes.h"
+
+#include <vtkSmartPointer.h>
+
+class vtkMatrix4x4;
+
+class AtracsysTracker
+{
+public:
+  /* Constructor & destructor */
+  AtracsysTracker();
+  virtual ~AtracsysTracker();
+
+  enum ATRACSYS_RESULT
+  {
+    SUCCESS = 0,
+    ERROR_UNABLE_TO_GET_FTK_HANDLE,
+    ERROR_NO_DEVICE_CONNECTED,
+    WARNING_CONNECTED_IN_USB2,
+    ERROR_UNABLE_TO_LOAD_MARKER,
+    ERROR_FAILURE_TO_LOAD_INI,
+    ERROR_OPTION_AVAILABLE_ONLY_ON_FTK,
+    ERROR_OPTION_AVAILABLE_ONLY_ON_STK,
+    ERROR_FAILED_TO_CLOSE_SDK,
+    ERROR_CANNOT_CREATE_FRAME_INSTANCE,
+    ERROR_CANNOT_INITIALIZE_FRAME,
+    ERROR_NO_FRAME_AVAILABLE,
+    ERROR_INVALID_FRAME,
+    ERROR_TOO_MANY_MARKERS,
+    ERROR_ENABLE_IR_STROBE,
+    ERROR_SET_USER_LED,
+    ERROR_ENABLE_USED_LED,
+    ERROR_SET_MAX_MISSING_FIDUCIALS,
+    ERROR_ENABLE_ONBOARD_PROCESSING,
+    ERROR_ENABLE_IMAGE_STREAMING,
+    ERROR_ENABLE_WIRELESS_MARKER_PAIRING,
+    ERROR_ENABLE_WIRELESS_MARKER_STATUS_STREAMING,
+    ERROR_ENABLE_WIRELESS_MARKER_BATTERY_STREAMING
+  };
+
+  enum DEVICE_TYPE
+  {
+    UNKNOWN_DEVICE = 0,
+    SPRYTRACK_180 = 1,
+    FUSIONTRACK_500 = 2,
+    FUSIONTRACK_250 = 3
+  };
+
+  class Marker
+  {
+  public:
+    Marker(int geometryId, vtkSmartPointer<vtkMatrix4x4> toolToTracker, int gpm, float freMm);
+    int GetGeometryID();
+    int GetGeometryPrecsenceMask();
+    vtkSmartPointer<vtkMatrix4x4> GetTransformToTracker();
+    float GetFiducialRegistrationErrorMm();
+  private:
+    int Id; /*!< Tracking id */
```

Comment:
could you add a short comment explaining the difference between tracking and geometry id?